data/sql/updates/pending_db_characters/active_arena_season.sql

```diff
@@ -0,0 +1,7 @@
+DROP TABLE IF EXISTS `active_arena_season`;
+CREATE TABLE `active_arena_season` (
+    `season_id` TINYINT UNSIGNED NOT NULL,
+    `season_state` TINYINT UNSIGNED NOT NULL COMMENT 'Supported 2 states: 0 - disabled; 1 - in progress.'
+);
```

Comment:
Is it intended not to have a season_id not to have a key?

```suggestion
)
CHARSET = utf8mb4
COLLATE = utf8mb4_unicode_ci
ENGINE = InnoDB
;
```

Reply:
Yeah, it's a single row table, so primary key is not that useful.